static/src/javascripts/bootstraps/common.js

```diff
@@ -446,8 +452,17 @@ define([
                     && ab.testCanBeRun('MtStickyBtm')) {
                     stickyTest.fireStickyBottom();
                 }
-            }
+            },
 
+            internationalSignposting: function () {
+                if ('internationalEdition' in config.page) {
+                    if (config.page.internationalEdition === 'international') {
```

Comment:
'Cuz otherwise the `else` would also catch people aren't international at all. We don't want to show the opt in message to people who are in the UK or US for instance.